bgpd/bgp_route.c

```diff
@@ -9998,6 +10009,34 @@ static int bgp_show_table(struct vty *vty, struct bgp *bgp, safi_t safi,
 						"Route Distinguisher: %s\n",
 						rd);
 			}
+
+			if (use_json) {
+				/* encode prefix */
+				if (dest_p->family == AF_FLOWSPEC) {
+					char retstr
+						[BGP_FLOWSPEC_STRING_DISPLAY_MAX];
+
+					bgp_fs_nlri_get_string(
+						(unsigned char *)dest_p->u
+							.prefix_flowspec.ptr,
+						dest_p->u.prefix_flowspec
+							.prefixlen,
+						retstr, NLRI_STRING_FORMAT_MIN,
+						NULL, family2afi(dest_p->u
+							.prefix_flowspec.family));
+
+					json_object_array_add(
+						json_paths,
+						json_object_new_string(retstr));
```

Comment:
You add a string here into an array, while it should be an object, right? For example:
```
"10.0.0.0/24": { ... }
```

Reply:
here as well. i could not add this as an object.
we can add this inside the function call, but it would be shown as one of the other entries